README.md

```diff
@@ -72,12 +53,64 @@ cd src/py3.9
 pipenv lock
 ```
 
-## New Repositories from a Skeleton ##
+## Lambda inputs ##
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| account\_ids | The list of AWS accounts to query for egress IPs to publish. | `list(string)` | n/a | yes |
+| application\_tag | The name of the AWS tag whose value represents the application associated with an IP address. | `string` | `"Application"` | no |
+| bucket\_name | The name of the S3 bucket to publish egress IP address information to. | `string` | n/a | yes |
+| domain | The domain hosting the published file(s) containing egress IPs. | `string` | `"example.gov"` | no |
```

Comment:
Perhaps overkill but it might be nice to specify that it should be a FQDN as the default value could lead to misinterpretation.